src/applications/discover-your-benefits/config/form.js

```diff
@@ -101,6 +104,38 @@ export const formConfig = {
           schema: militaryBranch.schema,
           depends: () => !environment.isProduction(),
         },
+        titleTenActiveDuty: {
+          path: 'service/active-duty',
+          title: 'Active Duty',
+          uiSchema: activeDuty.uiSchema,
+          schema: activeDuty.schema,
+          depends: formData => {
+            return (
+              !environment.isProduction() &&
```

Comment:
Technically, the production check isn't needed since the military branch question is required in order to get to this question, and that question is already hidden for production. But I thought I'd explicitly add this check here just to be safe for any future changes